website/content/docs/release-notes/1.12.0.mdx

```diff
@@ -0,0 +1,136 @@
+
+### Auto-fetch CRL in the Certificate Auth Method
+
+Operators will now be able to specify one or more CRL URLs that Vault will automatically fetch and keep up-to-date, rather than having to push the CRLs to the cert auth method. This should make certificate management easier for those users that have large cert auth deployments.
+
+### KMIP Server Profile
+
+The [Baseline Server Profile](https://docs.oasis-open.org/kmip/kmip-profiles/v2.1/os/kmip-profiles-v2.1-os.html) specifies the basic functionality expected of a KMIP server.  In Vault 1.12, we offer support for the operations and attributes in the Baseline server profile. With this release, Vault Enterprise now supports the Symmetric Key lifecycle server profile, Baseline server profile, and the Basic Cryptographic server profile (as of Release 1.11), enabling the  support of KMIP integrations with various clients more effectively. This requires the Enterprise ADP license.
+
+### GCP Cloud Key Manager Support
+
+Managed Keys let Vault secrets engines (currently PKI) use keys stored in Cloud KMS systems for cryptographic operations like certificate signing.  Vault 1.12 adds support for GCP Cloud KMS to the Managed Key system, where previously AWS, Azure, and PKCS#11 Hardware Security Modules were supported.
+
+### Path and Role-Based Resource Quotas
+
+In this release, the existing resource quota functionality has been enhanced. In addition to applying the API rate limiting and lease quotas at the namespace or mount level, you can now use  the quotas to the [API path suffixes and auth mount roles](/docs/enterprise/lease-count-quotas). This enhancement provides users with more control over issued certificates.
+
+### Client Count Improvements
+
+The billing period for client counting API can now be specified with the [current month](/docs/concepts/client-count) for the end date parameter. When this is done the "new_clients" field will have an hyperlog approximate value indicating the number of new clients that came in the current month. Note that for the previous months, the number will be an exact value.
+
+### Redis Secrets Engine
+
+With the support of the Redis Secrets Engine, users can use Vault to manage static or dynamic credentials for Redis OSS. The engine  works similarly to other database secrets engines. Refer to the [Redis](/docs/secrets/redis) documentation for more information. Huge thanks to [Francis Hitchens](https://github.com/fhitchen), who contributed their repository to HashiCorp
+
+### AWS Elasticache Secrets Engine
+With the support of the AWS ElastiCache Secrets Engine, users may use Vault to manage static  for AWS Elasticache instances that use Redis as their backend. The engine will work similarly to other database secrets engines. Refer to the [elasticache](/docs/secrets/databases/rediselasticache) documentation for more information.
```

Comment:
static roles...missing the role part.

Reply:
Fixed ([commit](https://github.com/hashicorp/vault/pull/17237/commits/4583f44c3151389f3f470d6abe502b37739e04be))